src/nsidc/iceflow/itrf/converter.py

```diff
@@ -69,17 +71,50 @@ def transform_itrf(
             if not plate:
                 plate = plate_name(Point(chunk.longitude.mean(), chunk.latitude.mean()))
             plate_model_step = (
-                f"+step +inv +init={target_itrf}:{plate} +t_epoch={target_epoch} "
+                # Perform coordinate propagation to the given epoch using the
+                # provided plate motion model (PMM).  An example is given in the
+                # message of this commit:
+                # https://github.com/OSGeo/PROJ/commit/403f930355926aced5caba5bfbcc230ad152cf86
+                f"+step +init={target_itrf}:{plate} +t_epoch={target_epoch} "
```

Comment:
I've removed the `+inv` step here. I think this is correct!

The `+inv` operator reverses the operation, which I think was placed here by mistake. We want to use the target ITRF's plate motion model to do a coordinate propagation to the target epoch, not the reverse.

See the linked commit's message for an example, which does not utilize the `+inv` operator: https://github.com/OSGeo/PROJ/commit/403f930355926aced5caba5bfbcc230ad152cf86

Reply:
This is a complete guess, but maybe we put the `+inv` operator here because  we have it in the ITRF transformation step:

```
+step +inv +init={target_itrf}:{source_itrf}
```

We do this for the ITRF transformation step because the `+init=` syntax looks up an init file with the corresponding entry (see https://proj.org/en/9.3/resource_files.html#init-files). There is not an init file for every ITRF, and for `valkyrie` we're mostly looking at doing ITRF conversions to ITRF2014.

So our pipeline is often constructed to look like this:

```
+step +inv +init=ITRF2014:ITRF2008
```

This looks up the ITRF2014 data file and finds the TIRF2008 transformation parameters (parameters to go from ITRF2014 to ITRF2008). By adding the `+inv`, we reverse the transformation so that our ITRF2008 data are converted to ITRF2014.

Reply:
Here's a proj mailing list thread about the correct syntax for plate motion models: https://lists.osgeo.org/pipermail/proj/2022-February/010527.html

In the example, a set of coordinates at epoch 2014.0 are propagated to 2022.0 using the ITRF2014:EURA model:

```
echo 3496746.5664 743256.4272 5264477.0556 2014.0 | cct +init=ITRF2014:EURA +t_epoch=2022.0
```

This would seem to confirm that the `+inv` should indeed be removed!

But later, in this message from Even: https://lists.osgeo.org/pipermail/proj/2022-February/010530.html


> apply the ITRF2014:NOAM model using a target epoch of 2010 (hence the +inv, as naturally the transformation would use +t_epoch as the "source") for a point at observation epoch of 2022.1


```
echo -100 40 0 2022.1 | cct -d 8 +proj=pipeline \
     +step +proj=cart +ellps+GRS80 \
     +step +inv +init=ITRF2014:ITRF2000 \
     +step +inv +init=ITRF2014:NOAM +t_epoch=2010 \
     +step +proj=set +v_4=2010 \
     +step +init=ITRF2014:ITRF2000 \
     +step +inv +proj=cart +ellps+GRS80
```